src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -3292,6 +3314,9 @@ bool SvgLoader::read()
 {
     if (!content || size == 0) return false;
 
+    if (((uint32_t)viewFlag & (uint32_t)SvgViewFlag::Viewbox) &&
+        (fabsf(vw) <= FLT_EPSILON || fabsf(vh) <= FLT_EPSILON)) return false;
```

Comment:
IMHO, If this is a wierd case, we could print TVGLOG for hints.

Reply:
done